addon/components/step-manager/step.js

```diff
@@ -26,6 +27,18 @@ export default Component.extend({
     }
 
     this['register-step'](this);
+
+    // This is a hacky fix to make the addon work on Ember.js <= 2.8. The
+    // reason is that the VisibilitySupport mixin that is implemented in
+    // every component requires the component to be in the DOM. In particular,
+    // the line at https://github.com/emberjs/ember.js/blob/v2.8.0/packages/ember-views/lib/mixins/visibility_support.js#L46
+    // is failing.
+    //
+    // As the component is not rendered in the DOM, we don't need any of the
+    // functionality provided by VisibilitySupport. So the fix is "ok".
+    if (isFunction(this._toggleVisibility)) {
```

Comment:
I decided to drop "official" support for anything below 2.8.0 since that's the LTS that's supported by the core team right now. Is this "hack" required for 2.8.0 compatibility, or could it be removed before cutting the `v1.0.0` release of the add-on?